CHANGELOG.md

```diff
@@ -8,14 +8,6 @@ and this project adheres to
 
 ## [Unreleased]
 
-### Added
-
-- Added `--step / -s` to `collect` command for the CLI, takes either a single
-  step or comma separated steps or multiple values. E.G. `--step fetch-account`,
-  `--step fetch-account,fetch-users`, or `--step fetch-account --step fetch-users`.
-
-## 0.6.1 - 2020-04-21
-
 ### Fixed
 
 - Fixed issue with dependency graph failing to wait for step dependencies to
```

Comment:
@austin-rausch I think I messed this up in my branch. this fix is in 0.6.1, but the `setupRecording` addition is not 🤦 
Sorry for the confusion

Reply:
Yea, I just fixed it.  I think it was me resolving the conflict wrong.